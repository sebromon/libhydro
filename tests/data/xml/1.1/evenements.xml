<?xml version="1.0" encoding="UTF-8"?>
<!-- <hydrometrie xmlns="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1 http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1/sandre_sc_hydrometrie.xsd"> -->
<hydrometrie>
  <Scenario>
    <CodeScenario>hydrometrie</CodeScenario>
    <VersionScenario>1.1</VersionScenario>
    <NomScenario>Echange de données hydrométriques</NomScenario>
    <DateHeureCreationFichier>2010-02-26T10:05:00+03:00</DateHeureCreationFichier>
    <RefFichier>20100226160756_donnees_sac30-2010-02-26-15-05.xml</RefFichier>
    <Emetteur>
      <CdIntervenant schemeAgencyID="SANDRE">1520</CdIntervenant>
      <NomIntervenant>Service de Prévision des Crues (S.P.C.) Grand Delta</NomIntervenant>
      <CdContact>26</CdContact>
    </Emetteur>
    <Destinataire>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
    </Destinataire>
  </Scenario>
  <Donnees>
    <Evenements>
      <!-- Evenement 1 -->
      <Evenement>
        <CdSiteHydro>A00010101</CdSiteHydro>
        <CdContact>1</CdContact>
        <DtEvenement>1999-08-12T00:05</DtEvenement>
        <DescEvenement>Arrachement de l'échelle</DescEvenement>
        <TypPublicationEvenement>1</TypPublicationEvenement>
        <DtMajEvenement>2000-05-10T22:05</DtMajEvenement>
      </Evenement>
      <!-- Evenement 2 -->
      <Evenement>
        <CdStationHydro>Z853010101</CdStationHydro>
        <CdContact>8563</CdContact>
        <DtEvenement>2010-02-26T10:05:00+01:00</DtEvenement>
        <DescEvenement>Déplacement de la station de 22.5m</DescEvenement>
        <TypPublicationEvenement>100</TypPublicationEvenement>
        <DtMajEvenement>2011-01-13T10:05:00+00:00</DtMajEvenement>
      </Evenement>
    </Evenements>
  </Donnees>
</hydrometrie>
