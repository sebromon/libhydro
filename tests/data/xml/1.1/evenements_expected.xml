<?xml version="1.0" encoding="UTF-8"?>
<!-- <hydrometrie xmlns="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1 http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1/sandre_sc_hydrometrie.xsd"> -->
<hydrometrie>
  <Scenario>
    <CodeScenario>hydrometrie</CodeScenario>
    <VersionScenario>1.1</VersionScenario>
    <NomScenario>Echange de données hydrométriques</NomScenario>
    <DateHeureCreationFichier>2010-02-26T07:05:00</DateHeureCreationFichier>
    <!-- <RefFichier>20100226160756_donnees_sac30-2010-02-26-15-05.xml</RefFichier> -->
    <Emetteur>
      <CdIntervenant schemeAgencyID="SANDRE">1520</CdIntervenant>
      <!-- <NomIntervenant>Service de Prévision des Crues (S.P.C.) Grand Delta</NomIntervenant> -->
      <CdContact>26</CdContact>
    </Emetteur>
    <Destinataire>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
    </Destinataire>
  </Scenario>
  <Donnees>
    <Evenements>
      <!-- Evenement 1 -->
      <Evenement>
        <CdContact>1</CdContact>
        <CdSiteHydro>A0010101</CdSiteHydro>
        <DtEvenement>1999-08-12T00:05:00</DtEvenement>
        <DescEvenement>Arrachement de l'échelle</DescEvenement>
        <TypPublicationEvenement>1</TypPublicationEvenement>
        <DtMajEvenement>2000-05-10T22:05:00</DtMajEvenement>
      </Evenement>
      <!-- Evenement 2 -->
      <Evenement>
        <CdContact>8563</CdContact>
        <CdStationHydro>Z853010101</CdStationHydro>
        <DtEvenement>2010-02-26T09:05:00</DtEvenement>
        <DescEvenement>Déplacement de la station de 22.5m</DescEvenement>
        <TypPublicationEvenement>100</TypPublicationEvenement>
        <DtMajEvenement>2011-01-13T10:05:00</DtMajEvenement>
      </Evenement>
    </Evenements>
  </Donnees>
</hydrometrie>
